composer.json

```diff
@@ -136,19 +136,26 @@
     "spryker/company": "^1.0.0",
     "spryker/company-business-unit": "^1.0.0",
     "spryker/company-business-unit-data-import": "^0.1.0",
+    "spryker/company-business-unit-extension": "^1.0.0",
+    "spryker/company-business-unit-gui": "^1.0.0",
+    "spryker/company-business-unit-gui-extension": "^1.0.0",
     "spryker/company-data-import": "^0.1.0",
     "spryker/company-extension": "^1.0.0",
     "spryker/company-gui": "^1.0.0",
+    "spryker/company-gui-extension": "^1.0.0",
     "spryker/company-mail-connector": "^1.0.0",
     "spryker/company-role": "^1.0.0",
+    "spryker/company-supplier": "^1.0.0",
```

Comment:
Supplier modules should not be wired to the suite (they confront upcoming Merchant concept).

Reply:
But what about company business unit?

Reply:
Company Business Unit should be wired.